index.js

```diff
@@ -1,36 +1,47 @@
 var path = require("path");
 const express = require("express");
+const bodyParser = require("body-parser");
 const app = express();
 const port = process.env.PORT || 3000;
-import * as dotenv from "dotenv";
-dotenv.config();
+require("dotenv").config();
 
 app.use("/static", express.static(path.join(__dirname, "public")));
+app.use(bodyParser.urlencoded({ extended: true }));
```

Comment:
This is needed tot pull the data out of the POST request. So that `req.body` is populated in our `POST /login` route.